src/main/protobuf/ReplicatedDataMessages.proto

```diff
@@ -101,5 +101,15 @@ message PNCounterMap {
   repeated Entry entries = 2;  
 }
 
+message ORMultiMap {
+  message Entry {
+    required string key = 1;
+    required ORSet value = 2;
+  }
+  
+  required ORSet keys = 1;
+  repeated Entry entries = 2;  
+}
+
```

Comment:
Yes, `protobufGenerate`
It is defined in https://github.com/patriknw/akka-data-replication/blob/master/project/Protobuf.scala
Same thing as in Akka.

There is also plugin for that https://github.com/sbt/sbt-protobuf
I would probably try that if adding this to another project.